pkg/generator/helper/helper.go

```diff
@@ -135,6 +137,7 @@ func {{.CobraCmdFuncName}}() *cobra.Command {
 				log.SetErrorCategory(log.ErrorConfiguration)
 				return err
 			}
+			vaultClient = config.GlobalVaultClient()
```

Comment:
If the `preRun` part for some reason fails after this line, then the Vault token will never be revoked.